account_payment_return/tests/test_payment_return.py

```diff
@@ -42,11 +39,9 @@ def setUpClass(cls):
             {
                 "name": "Test income account",
                 "code": "INCOME",
-                "user_type_id": cls.env["account.account.type"]
-                .create(
+                "account_type": "asset_receivable".create(
```

Comment:
Remove this create

Reply:
Done. There is one last test failing. It's weird because invoice_payments_widget field's value is False once the payment return is confirmed.